test/runTest.sh

```diff
@@ -236,7 +237,7 @@ fi
 if [[ -z "${LINT}" && -z "${GTEST}" && -z "${PYTEST}" ]]; then
   runAllTests
 else
-  if [ -z "${PYTEST}" ] && [[ "${APP_HOST}" || "${TRANSACTION_COUNT}"  || "${THREAD_COUNT}" || "${WORKSPACE}" || "${COV}" || "${TEST_NAME}" ]]; then
+  if [ -z "${PYTEST}" ] && [[ "${APP_HOST}" || "${TRANSACTION_COUNT}"  || "${THREAD_COUNT}" || "${WORKSPACE}" || "${COV}" || "${TEST_NAME}" != "${PYTEST_DIR}" || "${APPS}" != "--apps=slowFixDecoderApp" ]]; then
```

Comment:
This is not fool proof. If the user passes the default args it won't detect. 
If we need to reliably do this, it's better to no initialize the variables and then assign default values only within the method, if the variables are uninitialized.